CHANGELOG.md

```diff
@@ -1,3 +1,191 @@
+## Release 0.14.0
+
+Highlights:
+
+This release comes with two main new features.
+  * Plugins: Now you can create your HTTP-based plugin to provide new metrics
+    and display them in Scope. You can read more about it and see some examples
+    [here](https://github.com/weaveworks/scope/tree/master/examples/plugins).
+  * Metrics on canvas: Metrics are now displayed on the nodes and not just on
+    the details panel, starting with CPU and memory consumption.
+
+Also, the performance of the UI has been considerably improved and the 100-node
+rendering limit has been lifted.
+
+
+New features and enhancements:
+- Metrics on canvas
+	[#1105](https://github.com/weaveworks/scope/pull/1105)
+	[#1204](https://github.com/weaveworks/scope/pull/1204)
+	[#1225](https://github.com/weaveworks/scope/pull/1225)
+- Scope Plugins
```

Comment:
> Above its 'Plugins', here 'Scope Plugins'.
Will use 'Probe plugins' consistently. Thanks

> I would also be tempted to not repeat MoC and Plugins here, and just add the issues/pr links to the above.
I think we should separate the tickets from the friendly summary.